### PR TITLE
fix: Command + H can't hide rio on macOS #240

### LINKED ISSUE #240: `Command + H` can't hide rio on macOS
`Command + H` can't hide rio on macOS which is the expected behavior

$> rio --version
rioterm 0.0.19

### PR DESCRIPTION
**Pull request by sonbui00**
Fix #240 
Command + H can't hide rio on macOS

### CODE REVIEW

**Comment by raphamorim** [comment]
Looks good @sonbui00 
Question: would you like to take the opportunity and implement the toggle full screen in another PR (#229)

**Comment by sonbui00** [comment]
@raphamorim Yes. let me check

File: rio/src/sequencer.rs [hunk 1]
```diff
@@ -331,6 +332,10 @@ impl Sequencer {
                                     route.window.winit_window.select_previous_tab();
                                 }
                             }
+                            #[cfg(target_os = "macos")]
```
Comment (raphamorim) [inline rio/src/sequencer.rs:335]: I think hide is universal (besides wayland) no?

Comment (raphamorim) [reply rio/src/sequencer.rs:335]: Technically would not need the target_os

Comment (sonbui00) [reply rio/src/sequencer.rs:335]: key binding for this is only for MacOS and underline function (hide_application()) also implement for Macos only
If remove target_os filter, I think it still can run normally due to no key bind to call this

Comment (raphamorim) [reply rio/src/sequencer.rs:335]: Ah I see, I realised now that you are using hide_application, I thought hide also would work with https://docs.rs/winit/latest/winit/window/struct.Window.html#method.set_visible set_visible (that works for almost all platforms)

Comment (raphamorim) [reply rio/src/sequencer.rs:335]: Maybe hide can be global and if isn’t macos then use set_visible instead

Comment (sonbui00) [reply rio/src/sequencer.rs:336]: @raphamorim just try set_visible in MacOS, it's different with hide function that expect:
- hide: can back by alt + tab or icon on dock
- set_visible: don't how to back, maybe need call set_visible(true) to take it back

If you want I can add set_visible for other platform, so someone can bind on this

Comment (raphamorim) [reply rio/src/sequencer.rs:336]: Hmmm i see, well let’s implement only macos then and later we can revist it

**Review by raphamorim — APPROVED** [review]

**Comment by raphamorim** [comment]
Thank you @sonbui00

**Pull request merged**